site/content/blog/2023-09-09-browser-extension/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Parsing .DS_Store files with ZAP"
+summary: "ZAP Spider can now probe and parse macOS' .DS_Store files."
+images:
+- https://www.zaproxy.org/blog/2023-09-09-browser-extension/images/zapgsoc.png
+type: post
+tags:
+- blog
+- browser extension
+- gsoc 2023
+date: "2023-09-09"
```

Comment:
OK to publish on Monday? Will give us more time to review and publish the client add-on.
I would expect it to get more visibility during a weekday as well 😁

Reply:
Yes sure